### ISSUE TITLE
Stream starts stuttering after a few minutes of watching

### ISSUE THREAD

**Issue by Fruktus** [flathub/com.discordapp.Discord#476]
Hi. When I join a stream (as a viewer) it begins stuttering after a few minutes showing the yellow exclamation sign `Network connection issues` (but the network is stable and audio works as expected). Exiting and re-joining the stream fixes the issue for another couple of minutes. I only noticed this in the Flathub version of Discord, in the browser everything works as expected.

My system info:
```
Discord: stable 341883 (fd167e2) Host 0.0.72 x64 Build Override: N/A Linux 64-bit (6.10.3-200.fc40.x86_64)
OS: Fedora Linux 40 (Workstation Edition) with Wayland
```
Let me know if I should provide anything else

**Comment by guihkx** [comment]
> I only noticed this in the Flathub version of Discord,

What browser? Also, I think the web version should be lighter than the desktop version because it has a lot less features.


Can you also see this stuttering while running the non-Flatpak package? You can test this by downloading and extracting the official `.tar.gz` package, and running the `Discord` binary directly, unsandboxed.

**Comment by Fruktus** [comment]
> What browser? Also, I think the web version should be lighter than the desktop version because it has a lot less features.

Firefox browser 129.0 (64-bit)

I'll check later, thank you

**Comment by guihkx** [comment]
>Firefox browser 129.0 (64-bit)

Yeah, Discord on desktop runs on Chromium, so I imagine that's also a contributing factor...

>I'll check later

Great, please share the results here when you do!

**Comment by Fruktus** [comment]
Hi! Sorry that it took some time, I have tried to reproduce this issue using the tar version `stable 344389 (9236275) Host 0.0.74 x64 Build Override: N/A Linux 64-bit (6.10.3-200.fc40.x86_64)`  but after quite some time the issue did not occur.

I know this isn't exactly super helpful, perhaps this is a problem with my system.
If I could provide anything else please let me know.

**Comment by guihkx** [comment]
Thanks for the follow up. Unfortunately, performance/network issues are always tricky to pinpoint, and so I have no suggestions for now because I never really experienced this problem.

**Comment by guihkx** [comment]
Hi, can you retest this with latest Discord?

**Comment by Fruktus** [comment]
Hi, regrettably I've switched the OS to Ubuntu so I don't have the way to reproduce this anymore :(

**Comment by guihkx** [comment]
0.0.87 is on Flathub now. Run `flatpak update` to update.

I'll close this for now, but if you still experience this problem only with Flatpak and not with the official .tar.gz package, just let me know and I'll reopen.
